blockstore/splitstore/splitstore.go

```diff
@@ -228,6 +228,8 @@ func (s *SplitStore) Has(cid cid.Cid) (bool, error) {
 		return true, nil
 	}
 
+	// Didn't we need to somehow ressurect everything under this in case we decide to GC? Or are
+	// we handling that in a followup patch.
```

Comment:
we do, but first we have to implement #6705.
So subsequent pr, TBD before code freeze for v1.11.1.

Reply:
Otherwise anyone syncing from us will blow the hotstore with old messages and stuff.

Reply:
I will create a follow up issue for this, as I am not sure we'll get to it before code freeze for v1.11.1 and we also need to think a bit more about it -- i know @raulk has opinions on the matter.

Reply:
follow up issue in https://github.com/filecoin-project/lotus/issues/6726